leap_c/task.py

```diff
@@ -39,7 +40,8 @@ def __init__(
             mpc (MPCSolutionModule): The Model Predictive Control planner to be used
                 for this task.
             collate_fn_map (dict[type, Callable]): A dictionary mapping types to collate
-                functions. If None, the default collate function map is used.
+                functions. If None, the default collate function map is used, which is
+                often sufficient for most tasks.
```

Comment:
I would write "which is sufficient for most tasks" (leave often).
We should also make explicitly clear here that we are using OUR default collate function which entails more than pytorch's default collate. Also maybe it would be nice to refer to https://pytorch.org/docs/stable/data.html#torch.utils.data.default_collate for people who do not know collate.